sirmordred/sirmordred.py

```diff
@@ -194,8 +194,9 @@ def execute_nonstop_tasks(self, tasks_cls):
         """
             Just a wrapper to the execute_batch_tasks method
         """
+        sleep_for = self.conf.get('sortinghat', 1)
```

Comment:
I think it will take `sleep_for = conf['sortinghat']` if it exists instead of `sleep_for = ['sortinghat']['sleep_for']`, am I right?

Reply:
yes, that statement isn't correct! what do you think about this one:
`sleep_for = self.conf['sortinghat']['sleep_for'] if self.conf.get('sortinghat', None) else 1`?

As `sleep_for` isn't an optional parameter (https://github.com/chaoss/grimoirelab-sirmordred/blob/master/sirmordred/config.py#L481), so the code below should be ok